source/examples/generated/functions/helloWithValue.test.snippet.test-with-mocking.js

```diff
@@ -0,0 +1,29 @@
+// import the function
+const greet = require("../../functions/helloWithValue");
+
+// wrap the mock in beforeEach/afterEach blocks to avoid
+// pollution of the global namespace
+beforeEach(() => {
+  // mock of context.values.get()
+  global.context = {
```

Comment:
I bet we could write a tool that introspects your app's config files and generates a mock for you - potential 🦨 idea!

Reply:
that would be cool. JS doesn't support introspection of functions AFAIK so this would be an interesting challenge.